### DISCUSSION TITLE
Why does the NavigationView of the second example in AnimatedIcon Page has a lighting animation?

### DISCUSSION THREAD

**Discussion by HO-COOH** [microsoft/WinUI-Gallery#847] [Q&A]
It's just this one `NavigationView` has an animation when expanding or compact. I found it interesting and looked into the code and found no animation nor style associated with it. How is this possible?

**Comment by marcelwgn** [comment]
Hi @HO-COOH, which animation are you exactly referring to? The animations you are seeing are all part of the v2 WinUI 2 styles that you get when installing the Microsoft.UI.Xaml NuGet packge.

**Reply by HO-COOH** [reply]
Sorry for not being clear enough, my bad. You can see this ONE specific `NavigationView` has a light flashing after expanding and compacted, no other `NavigationView` has such an effect

https://user-images.githubusercontent.com/42881734/150980111-cba73a96-3182-47b8-8273-4e5add5cadaa.mp4

**Reply by marcelwgn** [reply]
Oh that is very interesting, I am not experiencing this:
![Opening and closing a NavigationView](https://user-images.githubusercontent.com/16122379/151047946-7170cc36-7856-4a73-ace0-0743eea25d5e.gif)

I am fairly sure that this is a bug and not intentional.

**Reply by HO-COOH** [reply]
For more context, I can observe this lighting in both WinUI2 Control Gallery and in WinUI3 Control Gallery on Windows 11.
On Windows 10, this lighting appears in WinUI3 Control Gallery, but not in WinUI2 Control Gallery.

**Reply by marcelwgn** [reply]
I checked this on my Win11 VM and it seems that the behavior is indeed present there. I would still argue that this is not intentional, especially since this is not present on Windows 10. Maybe @gregwoo-microsoft or @StephenLPeters know more about this.

**Reply by gregwoo-microsoft** [reply]
Yes I am able to repro this as well. Thanks for filing this bug. However, I am not sure what is causing this white flash.
